### PR TITLE
Fixing when delete a recordes that has two or more file fields in the Model

### PR DESCRIPTION
**Pull request by silviolleite**
This bug was fixed and now you can to delete yours recordes with two or more file fields.

### CODE REVIEW

**Comment by josegonzalez** [comment]
Can you revert all your spacing changes?

File: .idea/cake_config_setting_v0_6.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (josegonzalez) [inline .idea/cake_config_setting_v0_6.xml]: Please delete this file.

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (josegonzalez) [inline .idea/misc.xml]: Delete this as well.

**Comment by josegonzalez** [comment]
Needs a test as well.

File: Model/Behavior/UploadBehavior.php [hunk 3]
```diff
@@ -1028,10 +1022,10 @@ protected function _getThumbnailName($configuredName, $usePrefixStyle) {
 		}
 
 		if ($usePrefixStyle) {
-			return '{size}_{filename}';
+			return '{size}{filename}';
```
Comment (josegonzalez) [inline Model/Behavior/UploadBehavior.php]: Why this change?

Comment (silviolleite) [inline Model/Behavior/UploadBehavior.php]: Im sorry Jose i use it to dont let a original copy on folder ... it was fixed now thx

**Comment by josegonzalez** [comment]
I will merge this when you add test coverage.

**Comment by josegonzalez** [comment]
@silviolleite will you be providing a testcase for this?

**Comment by josegonzalez** [comment]
@silviolleite *bump*

**Comment by silviolleite** [comment]
Ohh yes... im sorry i was busy with my job... i will do it soon

**Comment by josegonzalez** [comment]
No problem :)

**Comment by josegonzalez** [comment]
@jorisvaesen thoughts on this?

**Comment by josegonzalez** [comment]
@silviolleite *bump*
